lib/cookies.server.ts

```diff
@@ -0,0 +1,29 @@
+import "server-only";
+
+import { cookies } from "next/headers";
+import {
+	ACCESS_TOKEN_KEY,
+	REFRESH_TOKEN_KEY,
+	TEN_DAYS_ms,
+} from "./cookies.client";
+
+export const accessTokenCookie = createCookieStorage(ACCESS_TOKEN_KEY);
+export const refreshTokenCookie = createCookieStorage(REFRESH_TOKEN_KEY);
+
```

Comment:
in which file did you set the accessTokenCookie in the server?
I see you set accessTokenCookie for the client during the login process but didn't find where you set accessTokenCookie in the server

Reply:
Cookie doesn't live in the server. They live in the browser and sent automatically with every request to the server. Cookies are used to share settings or state between the server and the client and they are stored in the browser.

So, we can change directly the cookies in the broswer.

When you set a cookie in the server, what actually happen is that the response tells the browser to modify its cookies. So here the browser again is the storage of these cookies.